modules/local/concat_h5ad.nf

```diff
@@ -0,0 +1,28 @@
+process CONCAT_H5AD {
+    label 'process_medium'
+
+    conda (params.enable_conda ? "conda-forge::scanpy conda-forge::python-igraph conda-forge::leidenalg" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/scanpy:1.7.2--pyhdfd78af_0' :
+        'quay.io/biocontainers/scanpy:1.7.2--pyhdfd78af_0' }"
+
+    input:
+    path h5ad
+    path samplesheet
+
+    output:
+    path "*.h5ad", emit: h5ad
+
+    script:
+    """
+    concat_h5ad.py \\
+        -i $samplesheet \\
+        -o combined_matrix.h5ad \\
+        -s "_matrix.h5ad"
```

Comment:
In general, I'd prefer the long-form arguments (`--input`) everywhere as it makes it much easier to understand what the parameters do without having to trace them back to the python script.

Reply:
Done in commit https://github.com/nf-core/scrnaseq/pull/117/commits/9ca1ee7a4557562a4e5f3e7c75a991781a149fd7